src/services/helpers/service.js

```diff
@@ -1,5 +1,5 @@
 const request = require('request-promise-native');
-const { GeneralError, Unavailable } = require('@feathersjs/errors');
+const { GeneralError, MethodNotAllowed } = require('@feathersjs/errors');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to resolve path to module '@feathersjs/errors'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4800486)